stylesheets/_get-started.scss

```diff
@@ -256,3 +256,41 @@ ul.toc {
 
 #new-rating, #your-rating, #avg-rating {display:none;}
 #rating-section {padding-top:1em;}
+
+.bpm-suite{
+  .ulist{
+    ul{
+      margin-bottom: 1.25rem;
+      li {
+      padding-left: 10px;
+      text-indent: -1em;
+      }
+    }
+  }
+
+  table{
+    background-color: transparent;
+    border: none;
+
+    tr{
+      background-color: transparent;
+    }
+    td{
+        border: none;
+      }
+    .ulist {
+      ul{margin-bottom: 0;}
+    }
+  }
+  h2{font-size: 1.6875rem;}
+}
+
+// List second line indentation
+.list-indent{
+  padding-bottom: 10px;
+
+  li {
+    padding: 10px 0 0 20px;
+    text-indent: -1em;
+  }
+}
```

Comment:
You should make sure files end with a new-line character. Otherwise some tooling and editors can get confused. Just putting an empty line at the end does the trick. GitHub will warn you of this, with the red icons you see on this line.